src/Components/Index.jsx

```diff
@@ -17,7 +17,7 @@ function Index() {
           firstH1={'Ready to start actually sticking to your goals?'}
           span={'Sign up for our free ebook on the most effective tatics for optimising your life:'}
          />
-        <Footer />
+        <Footer page={'HOME'} />
```

Comment:
?
If you will use router and high level handlers you don't need this weird declaration